src/content/developers/docs/frameworks/index.md

```diff
@@ -95,6 +95,12 @@ Before diving into frameworks, we recommend you first read through our introduct
 - [GitHub](https://github.com/chainstack)
 - [Discord](https://discord.gg/BSb5zfp9AT)
 
+**Crossmint -** **_The leading enterprise-grade web3 development platform, that allows you to build NFT applications without requiring any blockchain experience or holding cryptocurrency, and making the blockchain invisible to end users. The platform offers a one-stop-shop for most of your NFT infrastructure needs across all major chains EVM Chains(and others), including creating, distributing, selling, storing digital assets, and offering post-mint utility._**
```

Comment:
Mind trimming this description down to be more comparable in length to the others?

Reply:
I just updated it @wackerow